plugins/processors/execd/README.md

```diff
@@ -0,0 +1,73 @@
+# Execd Processor Plugin
+
+The `execd` processor plugin runs an external program as a separate process and pipes metrics in to the process's STDIN and reads processed metrics from its STDOUT. 
+The programs must output metrics in any one of the accepted 
+[Processor Data Formats](https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_INPUT.md) on its standard output.
+
+The `signal` can be configured to send a signal the running daemon on each collection interval.
+
+Program output on standard error is mirrored to the telegraf log.
+
+### Configuration:
+
+```toml
+[[processor.execd]]
+  ## Program to run as daemon
+  command = ["telegraf-smartctl", "-d", "/dev/sda"]
+
+  ## Delay before the process is restarted after an unexpected termination
+  restart_delay = "10s"
+
+  ## Data format your plugin will consume AND output. 
+  ## Must be supported by both a serializer and a parser!
```

Comment:
Make sure to update this, I think we need to split these.

Reply:
needs more thought. At the moment I would change this to say it's not possible to set the parser/serializer... which isn't ideal.. but as far as i know no parser/serializer combo would work except influx